src/agent/onefuzz-agent/src/local/common.rs

```diff
@@ -142,17 +130,56 @@ pub fn add_common_config(app: App<'static, 'static>) -> App<'static, 'static> {
 }
 
 fn get_uuid(name: &str, args: &ArgMatches<'_>) -> Result<Uuid> {
-    match value_t!(args, name, String) {
-        Ok(x) => Uuid::parse_str(&x)
-            .map_err(|x| format_err!("invalid {}.  uuid expected.  {})", name, x)),
-        Err(_) => Ok(Uuid::nil()),
-    }
+    value_t!(args, name, String).map(|x| {
+        Uuid::parse_str(&x).map_err(|x| format_err!("invalid {}.  uuid expected.  {})", name, x))
+    })?
+}
+
+pub fn get_synced_dirs(
+    name: &str,
+    job_id: Uuid,
+    task_id: Uuid,
+    args: &ArgMatches<'_>,
+) -> Result<Vec<SyncedDir>> {
+    let current_dir = std::env::current_dir()?;
+    let dirs: Result<Vec<SyncedDir>> = value_t!(args, name, PathBuf)?
+        .iter()
+        .enumerate()
+        .map(|(index, remote_path)| {
+            let path = PathBuf::from(remote_path);
+            let remote_path = path.absolutize()?;
```

Comment:
the `absolutize` doesn't do what I would expect when it comes to reports.

I ran the following:
`./target/debug/onefuzz-agent local libfuzzer --crashes_dir /tmp/local-test/crashes  --inputs_dir /tmp/local-test/inputs --unique_reports_dir /tmp/local-test/unique   --target_exe ./test.exe`

In the crash reports, this set the input blob to: 
```
  "input_blob": {
    "account": null,
    "container": null,
    "name": "tmp/local-test/crashes/crash-dc57cfd90a06ff5647a255718a2684209851a6bc"
  }
```

I would expect this to be `/tmp/local-tests/crashes/crash-...`.

Additionally, I would expect if I said `--crashes_dir crashes`, the name to be `crashes/crash-...`.  Instead, it ends up being the full path to $CWD/crashes minus the leading slash.

I think we can probably merge this PR and address this with a future PR.

Reply:
as discussed this will be addressed in a separate PR